Compacting Proposal (step one) (#18)
transmute-industries/ietf-spice-charter: charter.md

## Introduction

> Digital credentials are essential to identity, access control, and digitization use cases that are part of modernization efforts, such as digital licenses or certificates of origin.

Reviewer: The word "digitization" jumped out at me, and confused me.  I think of this as meaning to turn a book or music into an online form, and it wasn't obvious to me that this means turning licenses into online forms.  Is there another term, or is there a gentler way to get here, perhaps by speaking about licenses and certificates first?

Reviewer 2: what do you think about https://github.com/transmute-industries/ietf-spice-charter/pull/18/files#r1447421805

> Digital credentials are essential to identity, access control, and digitization use cases that are part of modernization efforts, such as digital licenses or certificates of origin.

Reviewer 2: ```suggestion
Digital credentials are essential to identity, authorization, licenses, certificates and digitization use cases that are part of modernization efforts targeting efficiency, transparency, such as digital licenses or certificates of origin.
```

> COSE and JOSE provide building blocks, such as identifiers and public and private claim structures, which enable interoperability and cryptographic agility, but which require profiling to render digitization easy, unambiguous and consistent.

Reviewer: "private claim structures" could refer to the fact that they can encrypted, or could refer to them being non-standard.

Reviewer 2: Since we refer to JOSE / COSE so much, I assumed it would refer to the registry topic.
A public claim is a registered one, private claims are not registered, and there are reserved blocks in COSE for them.

although I think it is helpful to leave room for addressing encryption use cases more thoroughly (as in not making encryption out of scope).

Reviewer: ```suggestion
IETF WGs and IRTF RGs have provided COSE and JOSE based building blocks.  Things like identifiers, claim structures, and confidentiality for personally identified information enable interoperability and cryptographic agility, however, they have required profiling in order to render digitization easy, unambiguous and consistent.
```

Reviewer 2: On the call, henk mentioned wanting to include some business cases beyond PII.

Author: lgtm (please expect some editorial changes though later on)

> There are various ways to combine identifiers, key material, signatures, and attributes to create credentials, which often prevents interoperabtility from spanning across data silos or cross use cases.

Reviewer: ```suggestion
The many ways to combine identifiers, key material, signatures, and attributes to create credentials, often prevents interoperabtility from spanning across data silos or cross use cases.
```

Reviewer 2: thanks!

> This lack of interoperabilty leads to increased implementation costs, attack surface, and harms adoption, which prevents digitzation from being deployed successfully.

Reviewer: ```suggestion
This lack of interoperabilty leads to increased implementation costs, attack surface, and harms adoption, which prevents digitization from being deployed successfully.
```

Reviewer 2: thanks!

SPICE aligns the capabilities of JOSE and COSE to support modern cryptographic capabilities, such as selective disclosure or unlinkablilty, consistent identity attribute semantics across representations, and Internet-scale discoverability of identifiers and cryptography capabilities.

## Background

> Foundational building blocks have been developed with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions, or other Selective-Disclosure and data minimization techniques.

Reviewer: Developed by whom/where?

Reviewer 2: I think in a charter, we don't need references for these but all of the listed ones here have been developed at CFRG or Privacy Pass or OAuth.

> Foundational building blocks have been developed with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions, or other Selective-Disclosure and data minimization techniques.

Reviewer 3: ```suggestion
IETF and IRTF working groups have developed foundational building blocks with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions, or other Selective-Disclosure and data minimization techniques.
```

While these techniques are being incorporated into JOSE and OAuth-based protocols, the integration into protocols using CBOR/COSE is, however, missing. 
The SPICE WG aims to close this gap by enabling the secure and privacy-friendly use of CBOR-based credentials for use cases that go beyond constrained Internet of Things devices. Envisioned use cases where these credentials will be used include education, digital wallets, business-to-business supply chain interactions, and digital media.

### A note on terminology (Note: context to be removed from charter upon adoption)

The term "digital credential" is a generic industry term, which does not imply a specific serialization. 

The term "verifiable credential" is a term which can imply, for example, JSON, JSON-LD or mDoc serialization.

In the W3C, "verifiable credential" implies specific JSON-LD based media types.

A "digital credential" is a "W3C Verifiable Credential" when it secures a JSON claimset that conforms to the W3C Technical Recommendation.

> A "digital credential" is an "OAUTH Verifiable Credential" when it secures a JSON claimset that conforms to the requirements of documents developed by the OAUTH WG. 

Reviewer 3: ```suggestion
A "digital credential" is an "OAuth Verifiable Credential" when it secures a JSON claimset that conforms to the requirements of documents developed by the OAuth WG. 
```
